### PR TITLE
Fix : golint - using copy() instead of a loop & go get deprecation

### PR DESCRIPTION
**Pull request by dilinade**
1. Fixed an issue in golint - using copy() instead of a loop
2. Replaced `go get` command with `go install` to add dependencies outside modules.

### CODE REVIEW

File: src/Dockerfile [hunk 1]
```diff
@@ -27,9 +27,8 @@ ENV GO111MODULE=on
 
 # Download protoc, protoc-gen-go, protoc-gen-go-grpc
 RUN apt-get update && apt-get install --no-install-recommends --assume-yes  protobuf-compiler \
-  && go get google.golang.org/grpc \
-  && go get google.golang.org/grpc/cmd/protoc-gen-go-grpc@latest \
-  && go get github.com/golang/protobuf/protoc-gen-go \
+  && go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@latest \
```
Comment (ustiugov) [inline src/Dockerfile:30]: reason for that change?

**Review by ustiugov — COMMENTED** [review]
Please provide the description of changes in the PR.

**Review by ustiugov — APPROVED** [review]

File: src/Dockerfile (hunk 1 above)
Comment (amariucaitheodor) [reply src/Dockerfile:30]: Thanks for this, but isn't the equivalent of _google.golang.org/grpc_ missing?

Comment (amariucaitheodor) [reply src/Dockerfile:30]: Thanks for this, but isn't the equivalent of google.golang.org/grpc missing?

**Review by amariucaitheodor — APPROVED** [review]

**Pull request merged**